packages/types/CHANGELOG.md

```diff
@@ -3,7 +3,50 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-# [0.14.0](https://github.com/cennznet/api.js/compare/v0.13.6...v0.14.0) (2019-06-18)
+
+## [UNRELEASED]
+    Changed:
+    - Improve getting started doc (#122)
+    Added:
+    - Updated Azalea metadata for runtime version 36 (#122)
+
+## [1.0.3](https://github.com/cennznet/api.js/compare/release/1.0.2...release/1.0.3) (13/07/2020)
+
+    Adds:
+    - Support CENNZX liquidity RPCs (#117)
+    - Slim metadata generation script (#120)
+    - Support on-chain asset registry (#116)
+    Fixes:
+    - Corrected some generated example snippets (#121)
```

Comment:
I think, this was after the release.. In the UNRELEASED section.. (corrected some generated example snippets)